Makefile

```diff
@@ -25,10 +32,11 @@ endif
 
 all: manager
 
-TESTS=$(shell go list ./... | grep -v /int | tr '\n' ' ')
+TESTTARGETS=$(shell go list ./... | grep -v /int | tr '\n' ' ')
```

Comment:
If you have to override this, we should make it overridable in boilerplate

Reply:
that is a future me problem